stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberElementUI.kt

```diff
@@ -39,6 +40,9 @@ import kotlinx.coroutines.job
 import kotlinx.coroutines.launch
 import com.stripe.android.core.R as CoreR
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+const val PHONE_NUMBER_TEXT_FIELD_TAG = "PhoneNumberTextField"
```

Comment:
Where is this being used? If it is not being used externally, can we keep this internal?

Reply:
It allows us to write test to programatically input a valid/invalid phone number and assert other states, such as [this](identity/src/test/java/com/stripe/android/identity/ui/IndividualScreenTest.kt)